libs/angular-remote-components/src/lib/components/slot/slot.component.ts

```diff
@@ -28,11 +28,8 @@ export class SlotComponent implements OnInit, OnDestroy {
   }
 
   subscription: Subscription | undefined
-  components$:
-    | Observable<
-        { componentType: Type<unknown> | undefined; remoteComponent: RemoteComponentInfo; permissions: string[] }[]
-      >
-    | undefined
+  components$: Observable<SlotComponentConfiguration[]> | undefined
```

Comment:
Change type to 'Observable<(SlotComponentConfiguration & {loading: boolean})[]>|undefined '

Use pipe(map(...)) to add the loading property here with starting value true